mne/io/egi/egimff.py

```diff
@@ -907,6 +909,142 @@ def _read_evoked_mff(fname, condition, channel_naming='E%d', verbose=None):
                        nave=nave, verbose=verbose)
 
 
+@fill_doc
+def export_evokeds_to_mff(fname, evoked, device, history):
+    """Export evoked dataset to MFF.
+
+    Use :func:`mne.export_evokeds` instead.
+
+    Parameters
+    ----------
+    %(export_params_fname)s
+    evoked : list of Evoked instances
+        List of evoked datasets to export to one file. Note that the
+        measurement info from the first evoked instance is used, so be sure
+        that information matches.
+    device : str
+        The device on which EEG was recorded (e.g. 'HydroCel GSN 256 1.0').
+    history : None | list of dict
+        Optional list of history entries (dictionaries) to be written to
+        history.xml. This must adhere to the format described in
+        mffpy.xml_files.History.content. If None, no history.xml will be
+        written.
+
+    Notes
+    -----
+    .. versionadded:: 0.24
+
+    Only EEG channels are written to the output file.
+    If no measurement date is specified, the current date/time is used.
+    """
+    info = evoked[0].info
+    if np.round(info['sfreq']) != info['sfreq']:
+        raise ValueError('Sampling frequency must be a whole number. '
+                         f'sfreq: {info["sfreq"]}')
+    sampling_rate = int(info['sfreq'])
+
+    # Initialize writer
+    mffpy = _import_mffpy('Export evokeds to MFF.')
+    import pytz
+    writer = mffpy.Writer(fname)
+    record_time = info['meas_date'] or \
+        pytz.utc.localize(datetime.datetime.utcnow())
+    writer.addxml('fileInfo', recordTime=record_time)
+    writer.add_coordinates_and_sensor_layout(device)
```

Comment:
this writer checks that device is a valid string? I foresee many typos in the device name otherwise

Reply:
Yes, the device string is used to lookup the corresponding generic coordinates and sensorLayout XMLs. If these are not found, an error is thrown.